nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1690,7 +1823,11 @@ async fn vpc_firewall_rules_get(
 }
 
 /**
- * Replace the firewall rules for a VPC
+ * Replace the firewall rules
+ *
+ * Replaces the firewall rules in a VPC with the specified rules. Any existing
+ * firewall rules are overwritten by the new rules.
+ * TODO: write a bit about the semantics of this operation since it is finiky.
```

Comment:
cc @davepacheco mostly looking for a blurb about what happens if the rules are changed before this API call is run and what if the etags differ I guess.